arachni/readme.md

```diff
@@ -5,31 +5,36 @@ scanner. Arachni is a feature-full, modular, high-performance Ruby framework
 aimed towards helping penetration testers and administrators evaluate the
 security of modern web applications.
 
-Web and REST API services are started on ports `9292` and `7331` respectively.
-API is not using any authentication either so use NGINX or some other proxy to
-add authentication if needed.
+Due to the difference in the way cli version of the scanner works over
+the API, this image is using a very simple web UI to launch the console
+scanner.
 
-## Database
+Frontend is written in [`Flask`](http://flask.pocoo.org/),
+[`Celery`](http://www.celeryproject.org/), and [`Redis`](https://redis.io/).
+Celery is using a single worker so only one instance of the scanner can run
+at the same time.
 
-By default if you do not pass any parameters to `docker run`, arachni will use
-local SQLite database. If you want to use Postgres, pass the following
-arguments:
+After the scan completion an email report is sent using Amazon SES. Because
+the html report is too big and other versions are not human-friendly, the
+scanner source code has been patched to allow plain text reports.
 
-* PG_HOST
-* PG_DATABASE
-* PG_USERNAME
-* PG_PASSWORD
+Web UI is running on port 5000.
 
-For example:
+## Running the image
 
+You need to pass the following environment variables:
+
+- `QA_EMAIL`
```

Comment:
I disagree with the naming of `QA_EMAIL`, `ROOT_EMAIL`, etc. This directly reflects UNIT9's current organisational structure and hardcodes a particular policy (cc qa@, bcc root@). The way these variables are used here, I would suggest renaming to `TO_EMAIL`, `BCC_EMAIL`.